Plugins/Event-Logs/Get-WindowsEvents.ps1

```diff
@@ -0,0 +1,287 @@
+<#
+
+.SYNOPSIS
+    Plugin-Name: Get-WindowsEvents.ps1
+    
+.Description
+    This plugin retrieves Windows event log entries based on the list of entries located
+    in the body of the script. Events that are not interesting can be ignored by 
+    commenting them out. Additional event types can be added to the list. The desired 
+    output for this plugin is CSV so that the events can be analyzed using filters.
+    Event IDs were gathered from multiple sources and some may not be relevant in all
+    scenarios. Make sure to validate the event IDs that you will be collecting.
+
+    Note: This plugin is designed for Windows Vista and Higher. Event IDs may be 
+    Different for XP and Server 2003 machines. I did my best to add in details on what
+    each event is for ease of tuning.
+
+    Note: If you plan to use this plugin with all event logs enabled there will be noise.
+    I would recommend tuning the event logs lists below to make sure they are applicable
+    to your scenario. Running this wide open is not a bad idea, but may be more difficult 
+    for newer analysts to quickly sift through the contents. I would recommend running,
+    finding a pivot point, and then search for events around your pivot point.
+
+.EXAMPLE
+    .\Get-WindowsEvents.ps1 -ComputerName Test-PC -Start_Time 12/23/2018
+
+    Power-Response Execution
+
+    Set ComputerName Test-PC
+    Set Start_Date 12/23/2018 
+    Run
+
+.NOTES
+    Author: 5yn@x
+    Date Created: December 23rd, 2018
+    Twitter:  @5ynax
+    
+    Last Modified By:
+    Last Modified Date:
+    Twitter:
+  
+#>
+
+param (
+
+    [Parameter(Mandatory=$true, Position=0)]
+    [string[]]$ComputerName,
+
+    [Parameter(Mandatory=$false, Position=1)]
+    [string]$Start_Date
```

Comment:
Could convert this to type `[DateTime]` with a default value of `(Get-Date).AddDays(-90)`

Also it looks like this is the only variable with a `_` character, to conform it to the rest of the script camel casing consider changing to `StartDate`